doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -54,6 +54,7 @@ Working with nRF54L Series
 ==========================
 
 * Added the :ref:`ug_nrf54l15_gs` page.
+* The default for Kconfig option :kconfig:option:`CLOCK_CONTROL_NRF_ACCURACY_PPM` has chaned from 500 to 250.
```

Comment:
There's a typo in `chaned`, so let's make an edit to the whole line -- please apply it also to line 67.
```suggestion
* Changed the default value for the Kconfig option :kconfig:option:`CLOCK_CONTROL_NRF_ACCURACY_PPM` from 500 to 250.
```